docs/databases/connections/athena.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Amazon Athena
+---
+
+# Amazon Athena
+
+Connecting Metabase to Athena depends on where Metabase is running.
+
+## Connecting to Athena from a Metabase Cloud instance
```

Comment:
I'm not sure if we need to add "Metabase Cloud" here...

The default method for most users will be using Access key + Secret key. So, this is the "default".

And then, there is an optional way of auth using AWS Credentials Chain.